dist/net.unvanquished.Unvanquished.appstream.xml

```diff
@@ -0,0 +1,85 @@
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/05/20210504-082658-002.unvanquished-0.52-human-builder.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2021/05/20210504-082658-002.unvanquished-0.52-human-builder-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2020/09/20200909-025733-001.unvanquished-rocket-missile.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2020/09/20200909-025733-001.unvanquished-rocket-missile-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/05/20210504-081116-002.unvanquished-0.52-alien-team.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2021/05/20210504-081116-002.unvanquished-0.52-alien-team-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2020/09/20200909-001900-000.unvanquished-lasgun-weapon.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2020/09/20200909-001900-000.unvanquished-lasgun-weapon-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/05/20210504-083923-002.unvanquished-0.52-alien-builder.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2021/05/20210504-083923-002.unvanquished-0.52-alien-builder-768x432.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://unvanquished.net/wp-content/uploads/2021/05/20210504-072808-002.unvanquished-0.52-human-team.jpg</image>
+      <image type="thumbnail" width="768" height="432">https://unvanquished.net/wp-content/uploads/2021/05/20210504-072808-002.unvanquished-0.52-human-team-768x432.jpg</image>
+    </screenshot>
+  </screenshots>
+
+  <url type="homepage">https://unvanquished.net</url>
+  <url type="bugtracker">https://github.com/Unvanquished/Unvanquished/issues</url>
+  <url type="help">https://wiki.unvanquished.net/wiki/Main_Page</url>
+
+  <releases>
+    <release version="0.52.0" date="2021-05-14">
+      <description>
+        <p>
+          The first beta release, coming after years of continuous work
+          both on the engine and the game itself
+        </p>
+      </description>
+      <url>https://unvanquished.net/unvanquished-0-52-beta-is-there/</url>
+    </release>
+  </releases>
+
+  <recommends>
+    <memory>3072</memory> <!-- 3GiB RAM -->
+  </recommends>
+
+  <requires>
+	  <control>pointing</control>
+  </requires>
+
+  <requires>
+	  <control>keyboard</control>
+  </requires>
```

Comment:
```suggestion
  <requires>
	  <control>pointing</control>
	  <control>keyboard</control>
  </requires>
```

Reply:
Would that mean both are needed?

Reply:
see https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-requires-recommends-control

Reply:
well, I've read it, three times, before making this PR, but apparently I got it wrong anyway ^^'